docs/Tutorials/Examples/Privacy-Example.md

```diff
@@ -147,7 +145,7 @@ Call [`eth_getTransactionReceipt`](../../Reference/API-Methods.md#eth_gettransac
 === "curl HTTP request"
 
     ```bash
-    curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["<TransactionHash>"],"id":1}' <JSON-RPC Endpoint>
+    curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["<TransactionHash>"],"id":1}' http://localhost:8545
```

Comment:
If the endpoint port is now necessarily 8545 maybe we have to add this to the requirements: do not have any service running on 8545 already

Reply:
Followed up with Ben to confirm if the quickstart fails gracefully